lib/containers/bats.pm

```diff
@@ -76,3 +79,16 @@ sub enable_modules {
     add_suseconnect_product(get_addon_fullname('sdk'));
     add_suseconnect_product(get_addon_fullname('python3'));
 }
+
+sub patch_logfile {
+    my ($log_file, @skip_tests) = @_;
+
+    foreach my $test (@skip_tests) {
+        # TODO: Remove "none" in openQA YAML schedules then next line
+        return if ($test eq "none");
+        if (script_run("grep -q 'in test file.*/$test.bats") != 0) {
```

Comment:
this grep looks wrong? `grep -q 'in test file.*/$test.bats` ? only one single quote?

Reply:
> this grep looks wrong? `grep -q 'in test file.*/$test.bats` ? only one single quote?

Nice catch!  Fixed.  Restarted VR.